### PR TITLE
On Android, allow PRAGMA statements that assign when invoked via `exec`

### PR DESCRIPTION
**Pull request by ilmoraunio**
Fixes the following error on Android

```
{"error": {"error": [Error: unknown error (code 0): Queries can be performed using SQLiteDatabase query or rawQuery methods only.], "insertId": undefined, "rows": undefined, "rowsAffected": undefined}}
```

when accessing `exec` directly to try to run assigning PRAGMA statements, such as `PRAGMA journal_mode = WAL`.

### CODE REVIEW

**Comment by craftzdog** [comment]
Would you add some tests to https://github.com/craftzdog/react-native-sqlite-2/blob/master/test/app.js?

**Comment by ilmoraunio** [comment]
Ah, will add.

**Comment by ilmoraunio** [comment]
Added some tests and also tested them on Android & iOS.

File: test/app.js [hunk 1]
```diff
@@ -67,7 +78,65 @@ export default class ReactNativeSQLite2Test extends Component {
...
+        resolve()
+      })
+    })
+  }
+
+  queryingPragma(db, isWal) {
+    new Promise((resolve) => {
+      let sql = "PRAGMA journal_mode"
+      db._db.exec([{sql: sql, args: []}], false, (_, result) => {
+        journal_mode = result[0].rows[0].journal_mode
```
Comment (craftzdog) [inline test/app.js:101]: Add `const` or it causes `ReferenceError`

File: test/app.js [hunk 2]
```diff
@@ -67,7 +78,65 @@ export default class ReactNativeSQLite2Test extends Component {
...
+        resolve()
+      })
+    })
+  }
+
+  assigningParenthesisPragma(db) {
+    new Promise((resolve) => {
+      let sql = "PRAGMA main.wal_checkpoint(FULL)"
+      db._db.exec([{sql: sql, args: []}], false, (_, result) => {
+        row = result[0].rows[0]
```
Comment (craftzdog) [inline test/app.js:127]: Add `const` or it causes `ReferenceError`

**Review by craftzdog — CHANGES_REQUESTED** [review]
Thanks! Can you fix them?

**Pull request merged**